### ISSUE TITLE
glossary entry for single or double quotes, would it be useful? 

### ISSUE THREAD

**Issue by anzonix** [python/devguide#592]
Single-, double-, or even, triple- quotes, quotation marks, may be confusing when used in examples in the glossary if they are not themselves listed in the glossary. 
People reading introductory material from other language backgrounds may worry, necessarily or unnecessarily.
It is stated generally, web-searching outside of the documentation, that in Python single and double quotation marks may be used interchangeably, as pairs,, but is this necessarily so?

Staying within the Python Developer’s Guide for information, on the Lexical Analysis page, 
"https://docs.python.org/3/reference/lexical_analysis.html#f-strings", at section 2.4.3
"he said his name is Fred" is an example where single and double quotations in the text and examples are presented.
The double and single quotes use, is partly explained with a good example further down that page ;-
"A consequence of sharing the same syntax as regular string literals is that characters in the replacement fields must not conflict with the quoting used in the outer formatted string literal:".

Also in PEP 498 , "https://www.python.org/dev/peps/pep-0498/", there is further explanation relating quotes. 
It may be that quotes do not require a glossary entry, but they do seem to require attention in their use.

**Comment by encukou** [comment]
Hello,
This repository contains the guide for contributing to Python, not for programming **in** Python. You can read this guide at https://devguide.python.org/ .
So, I'll close this issue here.

The Python documentation (https://docs.python.org) is part of the CPython repo: https://github.com/python/cpython/ .

On docs.python.org, string literals (and quotes) are covered: 
- in a friendly way in the tutorial: https://docs.python.org/3/tutorial/introduction.html#strings, and
- comprehensively & technically in Lexical analysis: https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

Unfortunately I myself don't have time right now to check the text. But if you find it inadequate, please file an issue at the tracker: https://bugs.python.org
But if you have any trouble with the processes, feel free to comment here.

**Comment by anzonix** [comment]
Hi Petr, 

Many thanks.
The informal introduction is helpful.

The lexical analysis itself in the text, at 2.4.1, does that very item of listing literals in double quotes and then referring to the same literals in single quotes, which although it does not contradict, "In plain English: Both types of literals can be enclosed in matching single quotes (') or double quotes (").", it does seem arbitrary, or perhaps proof of richness of the language.

Having fun with this herpetological language!
Regards, Toni.

________________________________________
